stackslib/src/config/mod.rs

```diff
@@ -2122,82 +2122,380 @@ impl NodeConfig {
 
 #[derive(Clone, Debug, PartialEq)]
 pub struct MinerConfig {
+    /// Time to wait (in milliseconds) before the first attempt to mine a block.
+    ///
+    /// Default: 10 ms
+    /// Deprecated: This setting is ignored in Epoch 3.0+. Only used in the neon chain mode.
     pub first_attempt_time_ms: u64,
+    /// Time to wait (in milliseconds) for subsequent attempts to mine a block,
+    /// after the first attempt fails.
+    ///
+    /// Default: 120_0000 ms (2 minutes)
+    /// Deprecated: This setting is ignored in Epoch 3.0+. Only used in the neon chain mode.
     pub subsequent_attempt_time_ms: u64,
+    /// Time to wait (in milliseconds) to mine a microblock,
+    ///
+    /// Default: 30_000 ms (30 seconds)
+    /// Deprecated: This setting is ignored in Epoch 3.0+. Only used in the neon chain mode.
     pub microblock_attempt_time_ms: u64,
-    /// Max time to assemble Nakamoto block
+    /// Maximum time (in milliseconds) the miner spends selecting transactions from the mempool
+    /// when assembling a Nakamoto block. Once this duration is exceeded, the miner stops
+    /// adding transactions and finalizes the block with those already selected.
+    ///
+    /// Default: 5000 ms (5 seconds)
     pub nakamoto_attempt_time_ms: u64,
-    /// Strategy to follow when picking next mempool transactions to consider.
+    /// Strategy for selecting the next transaction candidate from the mempool.
+    /// Controls prioritization between maximizing immediate fee capture vs. ensuring
+    /// transaction nonce order for account progression and processing efficiency.
+    ///
+    /// See [`MemPoolWalkStrategy`] for variant details.
+    ///
+    /// Possible values (use variant names for configuration):
+    /// - `"GlobalFeeRate"`: Selects the transaction with the highest fee rate globally.
+    /// - `"NextNonceWithHighestFeeRate"`: Selects the highest-fee transaction among those
+    ///   matching the next expected nonce for sender/sponsor accounts.
+    ///
+    /// Default: `"GlobalFeeRate"`
     pub mempool_walk_strategy: MemPoolWalkStrategy,
+    /// Probability (percentage, 0-100) of prioritizing a transaction without a known fee rate
+    /// during candidate selection.
+    ///
+    /// Only effective when `mempool_walk_strategy` is `GlobalFeeRate`. Helps ensure
+    /// transactions lacking fee estimates are periodically considered alongside high-fee ones,
+    /// preventing potential starvation. A value of 0 means never prioritize them first,
+    /// 100 means always prioritize them first (if available).
+    ///
+    /// Default: 25 (25% chance)
```

Comment:
```suggestion
    /// Default: `25` (25% chance)
```

added backticks to the value to match other comments

Reply:
one small change -  otherwise lgtm!